docs/Manual/Programs/Starter/Options.md

```diff
@@ -94,12 +94,15 @@ and pass it through the `--auth.jwt-secret-path` option.
 For example:
 
 ```bash
-echo "MakeThisSecretMuchStronger" > jwtSecret
+arangodb create jwt-secret --secret=jwtSecret
```

Comment:
will one also be able to use this only with commandline parameters and no files?  Ala: 

    arangodb create jwt-secret --secret=- < jwtSecretOpenSesame